firmware/src/util/battery.cpp

```diff
@@ -21,8 +21,8 @@
 
 #include <cmath>
 
-#define ADC_LINFIT_A 0.00818474f
-#define ADC_LINFIT_B 0.476469f
+#define ADC_LINFIT_A 0.00826849f
```

Comment:
I think you have a wolframalpha link with these two constants somewhere below, pls update it